docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md

```diff
@@ -49,6 +49,13 @@ spec:
 | hostName | string | The hostname or IP address of the remote git server. Default is the same value with Host. | No |
 | sshKeyFile | string | The path to the private ssh key file. This will be used to clone the source code of the specified git repositories. | No |
 | sshKeyData | string | Base64 encoded string of SSH key. | No |
+| personalAccessToken | [PipedGitPersonalAccessToken](#gitPersonalAccessToken) | Configuration for personal access token. | No |
```

Comment:
Just realized that;
PipeCD supports not only GitHub but also Git services(GitLab, self-host Git server, etc.).
So, it should be how this configuration can be used for other Git services.

How about simply adding `password` instead of `personalAccessToken`?
WDYT?

Reply:
Are you suggesting replacing the Personal Access Token with a password?
Using a password might lead to confusion, so maybe we could stick with some form of access token instead?

Reply:
@kentakozuka

Reply:
You are right. It can be confusing for GitHub users. But, IMHO, using the word password should be better because Personal access tokens can be used in GitHub (and GitLab as well), but it is not required for being a Git server. PipeCD supports Git (not GitHub) so it should respect Git terminology.

Reply:
In the case of 'Password', since it ends up overlapping with the name specified in 'UserName', I have designated 'PersonalAccessToken' as 'PasswordAuth' and 'userToken' as 'Password'.

Reply:
Since 'Password' overlaps with the variable name, 'PersonalAccessToken' has been renamed to 'PasswordAuth' and 'userToken' to 'Password'.

Reply:
@ffjlabo @kentakozuka 
I have modified the PAT settings to use a password, and updated it to work with the existing Git username!
How does this look?